nim_status/lib.nim

```diff
@@ -59,3 +68,17 @@ proc identicon*(str: string): string =
     result = generateBase64(str)
   except:
     discard
+
+
+
+##### TODO: move to somefile
+
+type StatusObject* = object
+  accountsDB*: DbConn
+  appDB*: DbConn
```

Comment:
I feel that this name is not very descriptive of what it contains unless you think that it may contain other properties in the future. We are describing an object of database connections, so instead could we call it `DatabaseConnections` or something similar? I guess if we are planning to add other properties to the object, then maybe we should keep the name.

Reply:
The name sucks but in my defense, this PR is something not mean to be merged as it is right now. It's still a WIP, but I'm pushing changes to the repo because this way it's easier to show/discuss what I'm doing

Reply:
Totally agree. No defence needed ☺️